documentation/WhitePapers/SustainabilityUseCasesAndLandscape2023.md

```diff
@@ -2,6 +2,8 @@
 
 This captures the known and ongoing sustainability efforts within the cloud native landscape as well as identifies challenge areas.
 
+All relevant wordings can be found explained here in the [glossary](https://github.com/cncf/tag-env-sustainability/blob/main/glossary/glossary.md).
```

Comment:
link should be https://tag-env-sustainability.cncf.io/glossary/

Reply:
here we go